### PR TITLE
Majur/internationalize hardcoded text

### PR DESCRIPTION
**Pull request by majur**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
I just internationalize some hardcoded texts

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

No UI changes

## Added/updated tests?

- [ ] Yes
- [X] No, and this is why: Nothing is changing
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by msarit** [comment]
@majur Hello! 👋🏾  Thank you for contributing to Forem!

Please resolve the conflicts, and address the comments made by @yheuhtozr. Let us know if you need any help! 🙏🏾

**Comment by citizen428** [comment]
@yheuhtozr If you have a moment, can you confirm that the changes in this PR (normal views, not admin views) are no longer necessary?

File: config/locales/fr.yml [hunk 1]
```diff
@@ -36,15 +37,18 @@ fr:
...
     month: Mois
     more_from: Plus de la part de
     my_tags: Mes sujets
+    other: Autre
     podcasts: Podcasts
     pinned: Épinglé
     popular_tags: Sujets populaires
     post: Publication
     posts: Publications
+    posted_on: Posté sur
```
Comment (yheuhtozr) [inline config/locales/fr.yml:51]: This translation seems not reflected on:
https://github.com/forem/forem/blob/43adc9f1fbe5667340dca39cc7dcd50741d7e486/config/locales/views/articles/fr.yml#L84

File: config/locales/fr.yml [hunk 2]
```diff
@@ -36,15 +37,18 @@ fr:
     loading: Chargement en cours...
     location: Lieu
     log_in: Se connecter
+    min_read: min lues
```
Comment (yheuhtozr) [inline config/locales/fr.yml:40]: This translation seems not reflected on:
https://github.com/forem/forem/blob/43adc9f1fbe5667340dca39cc7dcd50741d7e486/config/locales/views/articles/fr.yml#L86-L88
(might need to supply correct pluralization)

File: config/locales/fr.yml [hunk 3]
```diff
@@ -57,6 +61,8 @@ fr:
     reading_list: Liste de lecture
     reply: Répondre
     report_abuse: Signaler un abus
+    save: Enregistrer
+    saved: Enregistré
```
Comment (yheuhtozr) [inline config/locales/fr.yml:65]: The two lines not reflected on:
https://github.com/forem/forem/blob/43adc9f1fbe5667340dca39cc7dcd50741d7e486/config/locales/views/articles/fr.yml#L49-L50
(look a bit too long compared to English button width; we may hear native speakers if there are any remedies)

File: config/locales/fr.yml [hunk 4]
```diff
@@ -9,6 +9,7 @@ fr:
     all: Tout
     analytics: Analytique
     browse: Parcourir
+    code_of_conduct: Code de conduite
```
Comment (yheuhtozr) [inline config/locales/fr.yml:12]: This translation seems not reflected on:
https://github.com/forem/forem/blob/43adc9f1fbe5667340dca39cc7dcd50741d7e486/config/locales/views/articles/fr.yml#L16

**Review by yheuhtozr — COMMENTED** [review]
Follow-up review.

**Comment by yheuhtozr** [comment]
@citizen428 Hi, I have confirmed that all i18n replacements in this PR now invalidated, but most French translations they provided haven't been incorporated. Please take a look at my comments.

**Comment by citizen428** [comment]
@yheuhtozr Thanks a lot!
@majur Do you think you'll have time to make the necessary changes to this PR?

File: app/views/admin/navigation_links/index.html.erb [hunk 5]
```diff
@@ -34,7 +34,7 @@
 
   <div class="pb-8">
     <h2 class="mt-9 fw-bold fs-base">
-      Other
+      <%= t("core.other") %>
```
[the remark this answers is not in the document]
Comment (citizen428) [reply app/views/admin/navigation_links/index.html.erb:37]: Confirmed with the admin team, this is not the time.

**Comment by citizen428** [comment]
@yheuhtozr Just to be clear, do I understand correctly that the only things salvageable would be a few French translations?

**Comment by yheuhtozr** [comment]
@citizen428 Yes, IIRC. It contains source code i18n and locale file addition, and the former all overridden or nullified by our commits.

**Comment by citizen428** [comment]
Thanks @yheuhtozr, in this case, I'll close this PR.

**Pull request closed without merging**